pkgs/os-specific/bsd/netbsd/pkgs/make-rules.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  mkDerivation,
+  fetchNetBSD,
+  stdenv,
+  bsdSetupHook,
+  netbsdSetupHook,
+  rsync,
+}:
+
+mkDerivation {
+  path = "share/mk";
+  sha256 = "0w9x77cfnm6zwy40slradzi0ip9gz80x6lk7pvnlxzsr2m5ra5sy";
```

Comment:
does this hash need to be here?

Reply:
It does because that's NetBSD, which has many separate fetches (probably not good).